src/com/amazon/ion/impl/bin/Ion_1_1_Constants.java

```diff
@@ -8,32 +10,119 @@ private Ion_1_1_Constants() {}
 
     static final int FIRST_2_BYTE_SYMBOL_ADDRESS = 256;
     static final int FIRST_MANY_BYTE_SYMBOL_ADDRESS = 65792;
+    public static final int MAX_NANOSECONDS = 999999999;
+    public static final int NANOSECOND_SCALE = 9;
+    public static final int MAX_MICROSECONDS = 999999;
+    public static final int MICROSECOND_SCALE = 6;
+    public static final short MAX_MILLISECONDS = 999;
```

Comment:
Any particular reason why this one is a `short` instead of an `int`?

Reply:
Nope, changed